src/components/EventForm.js

```diff
@@ -4,8 +4,13 @@ import "react-dates/lib/css/_datepicker.css";
 import { DateRangePicker } from "react-dates";
 import moment from "moment";
 import uuid from "uuid";
+import PropTypes from "prop-types";
 
 export default class EventForm extends React.Component {
+  static defaultProps = {
```

Comment:
Please, refactor as discussed https://stackoverflow.com/questions/36778260/is-it-ok-to-put-proptypes-and-defaultprops-as-static-props-inside-react-class